doc/engineering-reference/src/daylighting-and-window-calculations/window-calculation-module.tex

```diff
@@ -392,6 +392,8 @@ \subsection{Glazing System Properties}\label{glazing-system-properties}
 
 where \({E_s}(\lambda )\) is the solar spectral irradiance function and \(V(\lambda )\) is the photopic response function of the eye. The default functions are shown in Table~\ref{table:solar-spectral-irradiance-function.} and Table~\ref{table:photopic-response-function.}. They can be overwritten by user defined solar and/or visible spectrum using the objects Site:SolarAndVisibleSpectrum and Site:SpectrumData. They are expressed as a set of values followed by the corresponding wavelengths for values.
 
+When a choice of Spectral is entered as optical data type, the correlations to store glazing system's angular performance are generated based on angular performance at 10 degree increments. When a choice of SpectralAndAngel is entered as optical data type, no correlations will be generated. The optical properties will be calculated by linear interpolation at a given incident angle.
```

Comment:
@mjwitte Here is my revision.

When a choice of Spectral is entered as optical data type, the correlations to store glazing system's angular performance are generated based on angular performance at 10 degree increments. When a choice of SpectralAndAngle is entered as optical data type, no correlations will be generated. The optical properties will be calculated by linear interpolation at a given incident angle to generate table data. The common incident angles are applied to all glass layers in the same construction. These table data of optical properties are used to generate polynomial curve fits with 6 coefficients as a function of cos of incident angles. The polynomial curves are used in simulations to calculate optical properties. 

Please let me know your opinion.